lib/fennec/evaluator/allocate/request.ex

```diff
@@ -175,8 +175,7 @@ defmodule Fennec.Evaluator.Allocate.Request do
   end
 
   defp udp_opts(server) do
-    ## TODO: {:active, true} is not an option for a production system!
-    [:binary, active: true, ip: server[:relay_ip]]
+    [:binary, active: UDP.Worker.burst_length(), ip: server[:relay_ip]]
```

Comment:
I guess there is `alias Fennec.UDP` missing in this file.

Reply:
☑️